### PR TITLE
👀 ORPO NLL Loss fix

### PR DESCRIPTION
**Pull request by Myashka**

### CODE REVIEW

File: turbo_alignment/trainers/dpo.py [hunk 1]
```diff
@@ -293,6 +293,10 @@ def compute_loss(
         ratio = -F.logsigmoid(log_odds)
         losses = self.beta * ratio
 
+        nll_loss = -policy_chosen_logps
+
+        losses += nll_loss
```
Comment (alekseymalakhov11) [inline turbo_alignment/trainers/dpo.py:298]: субъективно так приятнее
```suggestion
        losses = -policy_chosen_logps + self.beta * ratio
```

File: turbo_alignment/trainers/dpo.py [hunk 2]
```diff
@@ -657,9 +661,11 @@ def get_batch_metrics(
                 - torch.log1p(-torch.clamp(torch.exp(policy_rejected_logps), max=1 - 1e-7))
             )
             ratio = -F.logsigmoid(log_odds)
+            or_loss = self.dpo_loss_registry.beta * ratio
```
Comment (alekseymalakhov11) [inline turbo_alignment/trainers/dpo.py:664]: не могу оставить коммент там где надо

го еще то что сверху перепишем и уберем length_norm_policy_chosen_logps, потому что они уже у нас и там нормированы на длину и мы это второй раз делаем

**Review by alekseymalakhov11 — APPROVED** [review]
good job
